eva-server/src/main/resources/application.properties

```diff
@@ -17,4 +17,6 @@ spring.jmx.enabled = false
 springfox.documentation.swagger.v2.path=/webservices/api
 
 #spring.profiles.active=oauth2-security
-#security.oauth2.resource.user-info-uri = ...
\ No newline at end of file
+#security.oauth2.resource.user-info-uri = ...
+
+eva.mongo.collections.files=@eva.mongo.collections.files@
```

Comment:
I assume this parameter could be removed from `eva.properties` after all the WS that reference it are migrated?